### PR TITLE
Series was not already dropped, return false (and other fixes)

### PR DESCRIPTION
**Pull request by otoolep**
Code was not doing what comments said it should do.

### CODE REVIEW

**Comment by otoolep** [comment]
@pauldix

**Comment by toddboom** [comment]
@otoolep does this fix an open issue?

**Comment by otoolep** [comment]
No, this fixes no know issue, simply some drive-by fixes.

**Comment by jwilder** [comment]
LGTM :+1:

File: tsdb/meta.go [hunk 1]
```diff
@@ -408,7 +408,7 @@ func (m *Measurement) DropSeries(seriesID uint64) bool {
 		}
 	}
 
-	return true
+	return false
```
Comment (pauldix) [inline tsdb/meta.go]: is this return value used anywhere? Maybe it's better to update the method to not return anything if we're not actually using it.

Comment (otoolep) [inline tsdb/meta.go]: Let me check.

**Comment by pauldix** [comment]
+1

**Pull request merged**